themes/freemind/source/css/style.css

```
@@ -27,164 +27,208 @@ a {
 .post, .page {
   font-size: 1.2em;
   line-height: 1.5em;
-  font-family: 'IBM Plex Mono', monospace;
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)